Updates ubi.repo and RPMs to match with latest EC rules

pramodbindal opened the pull request:
Current rpms-lock contains some repos which are not supported as per latest EC rules. to fix that we need to update ubi.repo and rpm-locks  so that all the repos we refer are whitelisted in EC rules

Also added some dockerfile fixes

AndrienkoAleksandr commented:
Close. I moved this pr to upstream https://github.com/tektoncd/results/pull/268

Roming22 commented on pkg/api/server/v1alpha2/log/s3.go:
```diff
@@ -91,6 +103,32 @@ func NewS3LogStreamer(trl *v1alpha2.TaskRunLog, bufSize int, conf *conf.ConfigFi
+	if err != nil {
+		log.Printf("Failed to create bucket: %v", err)
+	}
+	return nil
```
Should not that return `err` instead of `nil` so that the error cascades down if the bucket could not be created?

pramodbindal committed:
Updates RPMS
Current rpms-lock contains some repos which are not supported as per latest EC rules. to fix that we need to update ubi.repo and rpm-locks  so that all the repos we refer are whitelisted in EC rules

Skip Prefetch

Regenerate RPMs

Regenerate RPMs

Update docker files

savitaashture commented on .tekton/tektoncd-results-main-watcher-pull-request.yaml:
```diff
@@ -37,9 +37,9 @@ spec:
+#  - name: prefetch-input
+#    value: |
+#      {"type": "rpm", "path": ".konflux/rpms"}
   pipelineRef:
```
This is required
